cuda_bindings/docs/source/release/13.X.Y-notes.rst

```diff
@@ -12,7 +12,9 @@ Released on TBD
 Highlights
 ----------
 
+* Migrated wheel dependencies from individual NVIDIA packages to the ``cuda-toolkit`` metapackage for improved dependency resolution and version constraints.
 * Automatic CUDA library path detection based on ``CUDA_HOME``, eliminating the need to manually set ``LIBRARY_PATH`` environment variables for installation.
+* The ``[all]`` optional dependencies now use ``cuda-toolkit`` with appropriate extras instead of individual packages. NVCC compiler is no longer automatically installed with ``pip install cuda-python[all]`` as it was previously included only to access NVVM library, which now has its own dedicated wheel. Users who need the NVCC compiler should explicitly install it with ``pip install nvidia-cuda-nvcc`` or the appropriate version for their needs.
```

Comment:
```suggestion
* The ``[all]`` optional dependencies now use ``cuda-toolkit`` with appropriate extras instead of individual packages. NVCC compiler is no longer automatically installed with ``pip install cuda-python[all]`` as it was previously included only to access NVVM library, which now has its own dedicated wheel. Users who need the NVCC compiler should explicitly install it with ``pip install cuda-toolkit[nvcc]==X.Y`` with the appropriate version for their needs.
```